server/kepler_model_trainer.py

```diff
@@ -174,7 +174,7 @@ def create_numerical_labels_weights_relation(numerical_labels, numerical_weights
 
 
 def create_categorical_vocab_weights_relation(categorical_labels, list_of_all_categorical_labels_vocab, list_of_all_categorical_labels_weights):
-    return {label: {list_of_all_categorical_labels_vocab[i][j]: {'weight': list_of_all_categorical_labels_weights[i][j]} for j in range(len(list_of_all_categorical_labels_vocab[i]))} for i, label in enumerate(categorical_labels)}
+    return {label: [{'name': list_of_all_categorical_labels_vocab[i][j], 'weight': list_of_all_categorical_labels_weights[i][j]} for j in range(len(list_of_all_categorical_labels_vocab[i]))] for i, label in enumerate(categorical_labels)}
```

Comment:
note: this affect changes in https://github.com/sustainable-computing-io/kepler/pull/225